client/me/billing-history/transactions-table.jsx

```diff
@@ -74,6 +74,10 @@ class TransactionsTable extends React.Component {
 			return <strong>{ this.props.translate( 'Multiple items' ) }</strong>;
 		}
 
+		if ( transactionItem.product === transactionItem.variation ) {
```

Comment:
Any possibility of these getting hitting funky edge cases like `null === null` or `undefined === undefined`? Should we make sure `transactionItem .product` is truthy?